app/src/ui/preferences/preferences.tsx

```diff
@@ -179,6 +183,9 @@ export class Preferences extends React.Component<
     const availableEditors = editors.map(e => e.editor)
     const availableShells = shells.map(e => e.shell)
 
+    const globalGitConfigPath = await getGlobalConfigPath()
+    this.setState({ globalGitConfigPath })
```

Comment:
No need to call `setState` twice, you can add `globalGitConfigPath` in the below `setState`